### DISCUSSION TITLE
Attempt to connect the orchestrator to the core failed. Error 400.

### DISCUSSION THREAD

**Discussion by Breno-Pimentel** [magma/magma#15361] [General]
That´s the problem:
scenario:
1. I created an orchestrator.
2. I created the core.
3. I changed the organization and DNS.
4. I updated the certificates.
5. I created the Gateway in the NMS.
After all these steps, Magma returns a status 400 when using the command "sudo service magma@* status."

Dec 28 18:19:35 core control_proxy[37565]: 2023-12-28T18:19:35.330Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.002s
Dec 28 18:19:36 core control_proxy[37565]: 2023-12-28T18:19:36.363Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.000s
Dec 28 18:19:36 core control_proxy[37565]: 2023-12-28T18:19:36.363Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.000s
Dec 28 18:19:37 core control_proxy[37565]: 2023-12-28T18:19:37.225Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.001s
Dec 28 18:19:37 core control_proxy[37565]: 2023-12-28T18:19:37.225Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.001s
Dec 28 18:19:38 core control_proxy[37565]: 2023-12-28T18:19:38.517Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.001s
Dec 28 18:19:39 core control_proxy[37565]: 2023-12-28T18:19:39.269Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.001s
Dec 28 18:19:40 core control_proxy[37565]: 2023-12-28T18:19:40.250Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.001s
Dec 28 18:19:40 core control_proxy[37565]: 2023-12-28T18:19:40.250Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.001s
Dec 28 18:19:40 core control_proxy[37565]: 2023-12-28T18:19:40.250Z [127.0.0.1 -> dispatcher-controller.magma.test,8443] "POST /magma.orc8r.SyncRPCService/EstablishSyncRPCStream HTTP/2" 400 215bytes 0.001s

**Comment by Breno-Pimentel** [comment]
It´s was fixed.

**Reply by panyogesh** [reply]
@Breno-Pimentel : Can you put the steps on how it was recovered.
